lib/generate-destination-data.js

```diff
@@ -0,0 +1,46 @@
+
+import getPathsAndTimes from './get-paths-and-times'
+import getTransitiveData from './get-transitive-data'
+
+export default function generateDestinationData ({
+  from,
+  inVehicleTravelTimes,
+  log = console.log.bind(console),
```

Comment:
It's just as default, mainly for the transition to using the WebWorkerPromiseInterface logging. We could just pass an empty function.